src/operations/finances.md

```diff
@@ -0,0 +1,67 @@
+# Finances
+
+The purpose of this document is to describe the financial status and processes of the Toronto Community Network.
+
+## Introduction
+The Toronto Community Network is a community project that current does not have its own independent legal entity or bank accounts, and relies on its fiscal hosting partners to provide fiscal administration capacity.
+Aside from out-of-pocket spendings from some of our members, our current financing comes from one single grant from the [Internet Society](https://www.internetsociety.org) administering through [Free Geek Toronto](https://www.freegeektoronto.org).
```

Comment:
Could be considered a false positive. It indicates instances of  spending.

However could also be written as such.

```suggestion
Aside from out-of-pocket spending from some of our members, our current financing comes from one single grant from the [Internet Society](https://www.internetsociety.org) administering through [Free Geek Toronto](https://www.freegeektoronto.org).
```

Reply:
"fiscal hosting partners" the prevalent term in the nonprofit sector is "trustee"